admin-manual/maintenance/maintenance.rst

```diff
@@ -343,39 +342,212 @@ edit a configuration file from the command line.
 2. With your preferred text editor, open the dashboard settings file (vim shown
    in example; Note that you must use sudo to edit this file):
 
-.. code:: bash
+   .. code:: bash
 
-    sudo vim /usr/share/archivematica/dashboard/settings/common.py
+      sudo vim /usr/share/archivematica/dashboard/settings/common.py
 
 3. Change the "DEBUG" flag from "False" to "True"
 
-.. code:: bash
+   .. code:: bash
 
-    DEBUG = True
+      DEBUG = True
 
 4. Save the fileservices
 
 5. Restart Apache
 
-.. code:: bash
+   .. code:: bash
 
-    sudo apache2ctl restart
+      sudo apache2ctl restart
```

Comment:
@mamedin is this still accurate? AFAIK we don't use Apache anymore?

Reply:
(this could be addressed separately of course)

Reply:
Also we would need to warn the user about this change, it should not be done in production. And we'd need to reload Gunicorn which can be done using a signal. I guess that I'll file an issue!

Reply:
We can address this as part of work on archivematica/Issues#81